praw/models/user.py

```diff
@@ -128,7 +128,7 @@ def me(
 
         """
         if self._reddit.read_only:
-            return None
+            raise RuntimeError("cannot operate in a userless context")
```

Comment:
Can you write "read_only" in the exception message?

Reply:
Is “`cannot operate in read-only mode`” better?

Reply:
Yes.